src/checkovRunner.ts

```diff
@@ -35,12 +34,11 @@ interface CheckovResponseRaw {
 
 const skipChecks = ['CKV_AWS_52'];
 
-export const runCheckovScan = (logger: Logger, checkovPath: string, extensionVersion: string, fileName: string, token: string, cancelToken: vscode.CancellationToken): Promise<CheckovResponse> => {
+export const runCheckovScan = (logger: Logger, extensionVersion: string, fileName: string, token: string, cancelToken: vscode.CancellationToken): Promise<CheckovResponse> => {
     return new Promise((resolve, reject) => {
         const checkovArguments: string[] = ['-s', '--skip-check', skipChecks.join(','), '--bc-api-key', token, '--repo-id', 'vscode/extension', '-f', `"${fileName}"`, '-o', 'json'];
-        const checkovCommand = `"${unescapePath(checkovPath)}"`;
-        logger.info('Running checkov', { checkovCommand, arguments: checkovArguments });
-        const ckv = spawn(checkovCommand, checkovArguments, 
+        logger.info('Running checkov', { arguments: checkovArguments });
```

Comment:
Now that I think about it, logging the token isn't a great practice...especially if we want them to share their log files in issues or slack

This is the most straight forward way

```
        const checkovLogArguments: string[] = ['-s', '--skip-check', skipChecks.join(','), '--bc-api-key', '****', '--repo-id', 'vscode/extension', '-f', `"${fileName}"`, '-o', 'json'];
        logger.info('Running checkov', { arguments: checkovLogArguments });
```
We can also map and change the token only 

```
        logger.info('Running checkov', { arguments: checkovLogArguments.map(argument => argument === token ? '****' : argument });
```

@elaygl WDYT?

Reply:
Sounds good